### ISSUE TITLE
How can i have the same gestures  for X11 that GNOME has by default in wayland

### ISSUE THREAD

**Issue by KonstantMan** [JoseExposito/touchegg#487]
Is there any way to have the 3 fingers  swipe up gesture to show the overview and then the app grid for X11 ?
I tried using the super key as a keyboard shortcut  but it doesn't work .

### Your environment

 * Version of Touchégg: Touchégg v2.0.9.
 * Operating System: Arch
 * Desktop Environment: Gnome X11

**Comment by JoseExposito** [comment]
Yes, have a look to https://github.com/JoseExposito/gnome-shell-extension-x11gestures
